client/trino-jdbc/src/main/java/io/trino/jdbc/TrinoStatement.java

```diff
@@ -161,7 +160,7 @@ public void setEscapeProcessing(boolean enable)
             throws SQLException
     {
         checkOpen();
-        escapeProcessing.set(enable);
+        // ignored
```

Comment:
Instead of a ignoring it. Can we throw an SQLException if it is enabled ? Like we don't support it as of now. 

cc: @electrum

Reply:
I looked around for how to handle this, and I found:

```
    @Override
    public void setCursorName(String name)
            throws SQLException
    {
        checkOpen();
        // ignore: positioned modifications not supported
    }
```

So I chose to ignore it here as well. But I'm open to alternative approaches.

Reply:
The difference might be of course that escape processing makes a change to the semantics of the query, so it should indeed complain loudly that this won't work as the user expects.